cli/cmd/helm/helmInstall.go

```diff
@@ -40,18 +40,20 @@ func addHelmInstallFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP("version", "", "", "Option to provide your tobs helm chart version, if not provided will install the latest tobs chart available")
 	cmd.Flags().BoolP("only-secrets", "", false, "Option to create only TimescaleDB secrets")
 	cmd.Flags().BoolP("skip-wait", "", false, "Option to do not wait for pods to get into running state (useful for faster tobs installation)")
+	cmd.Flags().BoolP("enable-prometheus-ha", "", false, "Option enable to prometheus and promscale high-availability by default scales to 3 replicas")
```

Comment:
"Option to enable prometheus and promscale high-availability, by default scales to 3 replicas"